metalearn/metafeatures/perceptron_metafeatures.py

```diff
@@ -0,0 +1,53 @@
+import numpy as np
+
+from sklearn.linear_model import Perceptron
+
+from metalearn.metafeatures.common_operations import profile_distribution
+
+
+def get_fitted_perceptron(X, Y, seed, n_classes, frac='all'):
+    class_fraction = n_classes/len(X)
+
+    if frac == 'all':
+        frac = class_fraction
+    elif frac == 'tenth':
+        frac = min(max(0.9, class_fraction), 1 - class_fraction)
+    elif frac == 'half':
+        frac = min(max(0.5, class_fraction), 1 - class_fraction)
+    elif frac == 'sqrt':
+        frac = min(max(1 - (np.sqrt(len(X)) / len(X)), class_fraction), 1 - class_fraction)
+
+    cls = Perceptron(random_state=seed, validation_fraction=frac, early_stopping=True, max_iter=1000, tol=1e-3)
```

Comment:
`cls` is usually used as the first argument to class methods, similar to `self`. See https://stackoverflow.com/questions/4613000/what-is-the-cls-variable-used-for-in-python-classes. It is a matter of style, but let's change this variable name anyways so its not confusing.

Reply:
Good call, I think I meant to use `clf` like they do in the sklearn docs but messed it up.